.buildkite/pipeline.yml

```diff
@@ -1,10 +1,13 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+---
+
+env:
+  IMAGE_ID: xcode-16.1-v4
```

Comment:
Could not upgrade to 16.4. See https://linear.app/a8c/issue/AINFRA-782/hostmgr-test-fails-in-xcode-164

Reply:
https://github.com/Automattic/hostmgr/pull/148

Reply:
🎉